src/llm/http_llm_calculator.cc

```diff
@@ -113,6 +121,16 @@ class OpenAIChatCompletionsRequest {
     }
 
     chat_t getMessages() const { return this->messages; }
+    Endpoint getEndpoint() const { return this->endpoint; }
```

Comment:
return const?

Reply:
The method already is const. Why do you want to return const Endpoint? It is copy anyway (integer) @mzegla

Reply:
It's a minor since this class is not widely used, but I was thinking that once set endpoint in the request should never be modified. But since it's a copy it doesn't really matter, so I'm okay with current version.